src/main/ruby/Gemfile

```diff
@@ -0,0 +1,18 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+source "https://rubygems.org"
+gem 'thrift', '0.11.0.0'
+gem 'accumulo', :path => "/path/to/accumulo-proxy/src/main/ruby/accumulo"
```

Comment:
Should this specify a ruby version also? Something similar to:

```
ruby '>2.6'
```

Reply:
I don't think this is done in the Gemfile.  I think it's done in the executable.

Reply:
I know it's possible to put that directive in the Gemfile, but not sure if it's necessary. I only know about it because I recently had to put that in our website's Gemfile for Travis.

Reply:
I put it in the Gemfile